app/scripts/services/session.js

```diff
@@ -7,13 +7,32 @@
  * @namespace hmdaPilotApp
  * @module {Service} Session
  */
-module.exports = /*@ngInject*/ function () {
+module.exports = /*@ngInject*/ function ($cookies, $cookieStore) {
 
-    var session = {
-        verifiedQualityEdits: [],
-        verifiedMacroEdits: {},
-        verifiedSpecialEdits: {},
-        verifiedIRSReport: false
+    var validPassword = 'p1l0t',
+        session = {
+            verifiedQualityEdits: [],
+            verifiedMacroEdits: {},
+            verifiedSpecialEdits: {},
+            verifiedIRSReport: false
+        },
+        loginDialogOptions = {
```

Comment:
Wouldn't hurt, but I put the progress bar config in the config file to keep it DRY. The config for the login dialog is only used once, so leaving it out of the config doesn't hurt anything.